docs/DesignMDIngestion.md

```diff
@@ -0,0 +1,126 @@
+# Metadata Ingestion from RING to Zenko
+
+## Description
+
+This is the design document discussing the ingestion process of existing metadata
+from the RING into Zenko with MongoDB.
+
+The primary use case is to copy the existing metadata from the RING/S3 Connector
+and ingest into the MongoDB backend that is used with Zenko so that users can
+connect existing buckets on RING to be managed by Zenko. This would allow users
+to complete operations such as metadata search and CRR. The ingestion process will
+also enable out-of-band updates. This will allow operations made directly to the RING
```

Comment:
This will allow operations made on ingested objects on Zenko to update the objects on RING.

Reply:
I thought it was both ways - Zenko manages the objects on the RING, but any changes/operations on the RING would also be reflected in Zenko

Reply:
Yup its both ways, but for OOB, it's being able to update the ingested metadata and having those changes reflect

Reply:
Oh wow lol, sorry i had a different def in my mind. Plz ignore my comments lol

Reply:
Np, ty, I should clarify that both functionalities are present here